src/System.Security.Cryptography.Primitives/ref/System.Security.Cryptography.Primitives.cs

```diff
@@ -14,10 +14,15 @@ public abstract partial class AsymmetricAlgorithm : System.IDisposable
         protected int KeySizeValue;
         protected System.Security.Cryptography.KeySizes[] LegalKeySizesValue;
         protected AsymmetricAlgorithm() { }
+        public virtual string KeyExchangeAlgorithm { get { throw null; } }
         public virtual int KeySize { get { throw null; } set { } }
         public virtual System.Security.Cryptography.KeySizes[] LegalKeySizes { get { throw null; } }
+        public virtual string SignatureAlgorithm { get { throw null; } }
+        public void Clear() { }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
+        public virtual void FromXmlString(string xmlString) { }
+        public virtual string ToXmlString(bool includePrivateParameters) { throw null; }
```

Comment:
Also, if we don't anticipate a successful return of these by .NET Core 1.2; we might  want Browsable(Never)